src/UXClient/Components/DateTimePicker/DateTimePicker.ts

```diff
@@ -108,6 +108,10 @@ class DateTimePicker extends ChartComponent{
         this.displayRangeErrors(rangeErrorCheck.errors);
```

Comment:
kinda was thinking that if this is created using a button, that "esc" should close it as well?  seems like a common flyout a11y pattern

Reply:
it totaly does